docs/en/interfaces/cli.md

```diff
@@ -67,22 +67,22 @@ The command-line client allows passing external data (external temporary tables)
 
 ### Queries with Parameters {#cli-queries-with-parameters}
 
-You can create a query with parameters, and pass values for these parameters with the parameters of the client app. For example:
+You can create a query with parameters and pass values of the corresponding client app parameters to them. For example:
```

Comment:
```suggestion
You can create a query with parameters and pass values to them from client application. This allows to avoid formatting query with specific dynamic values on client side. For example:
```